build.gradle.kts

```diff
@@ -2,34 +2,35 @@
 group = "no.nav.syfo"
 version = "1.0.0"
 
-val confluentVersion = "7.6.1"
+val confluentVersion = "7.8.0"
 val dialogmeldingVersion = "1.5d21db9"
 val fellesformat2Version = "1.0329dd1"
-val flywayVersion = "10.17.3"
-val hikariVersion = "5.1.0"
+val flywayVersion = "11.3.0"
+val hikariVersion = "6.2.1"
 val jacksonVersion = "2.18.2"
 val jaxbVersion = "2.3.1"
-val jsonVersion = "20240303"
-val kafkaVersion = "3.7.0"
+val jsonVersion = "20250107"
+val kafkaVersion = "3.9.0"
 val kithApprecVersion = "2019.07.30-04-23-2a0d1388209441ec05d2e92a821eed4f796a3ae2"
 val kithHodemeldingVersion = "2019.07.30-12-26-5c924ef4f04022bbb850aaf299eb8e4464c1ca6a"
 val kluentVersion = "1.73"
-val ktorVersion = "3.0.2"
-val logbackVersion = "1.5.12"
-val logstashEncoderVersion = "7.4"
-val micrometerRegistryVersion = "1.12.7"
-val mockkVersion = "1.13.13"
-val mqVersion = "9.3.4.1"
-val nimbusjosejwtVersion = "9.47"
-val postgresVersion = "42.7.4"
+val ktorVersion = "3.0.3"
```

Comment:
ktor 3.1.0 er visst ute nå

Reply:
Var ikke der da jeg sjekket for en time siden, men dukker opp nå. Da kan vi jo prøve ut den her 👍

Reply:
Ja, fant det ut nå nettopp når jeg skulle bumpe i en annen app :smile: